changed.d

```diff
@@ -359,33 +652,72 @@ void writeTextChangesBody(Entries, Writer)(Entries changes, Writer w, string hea
     }
 }
 
+bool lessImpl(ref BugzillaEntry a, ref BugzillaEntry b)
+{
+    if (!a.id.isNull() && !b.id.isNull())
+    {
+        return a.id.get() < b.id.get();
+    }
+    return false;
+}
+
+bool lessImpl(ref GithubIssue a, ref GithubIssue b)
+{
+    return a.number < b.number;
+}
+
+bool less(T)(ref T a, ref T b)
+{
+    return lessImpl(a, b);
+}
+
+enum BugzillaOrGithub {
+    bugzilla,
+    github
+}
+
 /**
 Writes the fixed issued from Bugzilla in the ddoc format as a single list.
 
 Params:
     changes = parsed InputRange of changelog information
     w = Output range to use
 */
-void writeBugzillaChanges(Entries, Writer)(Entries entries, Writer w)
+void writeBugzillaChanges(Entries, Writer)(BugzillaOrGithub bog, Entries entries, Writer w)
     if (isOutputRange!(Writer, string))
 {
     immutable components = ["DMD Compiler", "Phobos", "Druntime", "dlang.org", "Optlink", "Tools", "Installer"];
     immutable bugtypes = ["regression fixes", "bug fixes", "enhancements"];
+    const string macroTitle = () {
+        final switch(bog) {
+            case BugzillaOrGithub.bugzilla: return "BUGSTITLE_BUGZILLA";
+            case BugzillaOrGithub.github: return "BUGSTITLE_GITHUB";
+        }
+    }();
+    const string macroLi = () {
+        final switch(bog) {
+            case BugzillaOrGithub.bugzilla: return "BUGZILLA";
+            case BugzillaOrGithub.github: return "GITHUB";
```

Comment:
```
$(BUGSTITLE_GITHUB DMD Compiler bug fixes,

$(LI $(GITHUB 17118): GC not collecting under Windows)
)
$(BUGSTITLE_GITHUB Phobos bug fixes,

$(LI $(GITHUB 9094): std.traits: Unqual strips qualifiers on structs with references that would break const/immutable.)
)
$(BUGSTITLE_GITHUB dlang.org bug fixes,

$(LI $(GITHUB 3933): tightening auto decl spec)
)
$(BUGSTITLE_GITHUB Tools bug fixes,

$(LI $(GITHUB 425): coffimplib silently overwrites a library with a mixed-case extension)
)
)
```

The macro `GITHUB` is not specific enough, as each repository has a different url path.